public/i18n/en/translation.json

```
@@ -217,7 +217,13 @@
       "title": "Backup & restore"
     },
     "reboot_and_shutdown": {
-      "title": "Reboot & shutdown"
+      "title": "Reboot & shut down",
+      "reboot": "Reboot",
```

```suggestion
      "reboot": "Reboot unit",
```
(Mockup has just been updated)